docs/release-notes/NuGet-6.14.md

```diff
@@ -2,27 +2,130 @@
 author: zivkan
+ms.date: 5/6/2025
 ms.topic: conceptual
 ---
-# NuGet 6.14 Release Notes
 
-<!--
-    This is intentionally not yet added to TOC.md.
-    When this version is ready for release:
-    1. Rename this file to the next version
-    2. Change the version strings in this file to the next version
-    3. Use the release tool to create the real release notes for the version ready for release
-    4. Add the release notes to TOC.yml
--->
+# NuGet 6.14 Release Notes
 
 NuGet distribution vehicles:
 
 | NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
 |:---|:---|:---|
-| [**6.14**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.14](https://visualstudio.microsoft.com/downloads/) | [9.0.2xx](https://dotnet.microsoft.com/download/dotnet/9.0)<sup>1</sup> |
+| [**6.14.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.14.0](https://visualstudio.microsoft.com/downloads/) | [9.0.300](https://dotnet.microsoft.com/download/dotnet/9.0)<sup>1</sup> |
 
 <sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
 
-## Not yet released
+## Summary: What's New in 6.14.0
+
+* NuGet needs to onboard to Unified Settings and create General page - [#14040](https://github.com/NuGet/Home/issues/14040)
+
+* Support for new `slnx` solution format - [#14034](https://github.com/NuGet/Home/issues/14034)
+
+* dotnet-nuget-push now supports --config-file - [#4879](https://github.com/NuGet/Home/issues/4879)
+
+* `dotnet list package --vulerable` should support auditSources - [#13767](https://github.com/NuGet/Home/issues/13767)
+
+### Issues fixed in this release
+
+* Event tracing in new dependency resolver - [#14134](https://github.com/NuGet/Home/issues/14134)
+
+* Support for new slnx format in static graph-based restore - [#14086](https://github.com/NuGet/Home/issues/14086)
+
+* Add Net90 to FrameworkConstants.CommonFrameworks - [#14067](https://github.com/NuGet/Home/issues/14067)
+
+* Don&#39;t show usage help when dotnet cli command throws unhandled exception - [#14200](https://github.com/NuGet/Home/issues/14200)
+
+* Multiple callers check for NuGet entries before calling Error List `BringToFrontIfSettingsPermitAsync` - [#14163](https://github.com/NuGet/Home/issues/14163)
+
+* NuGetAudit should report transitive packages with vulnerabilities when targeting .NET 10 or higher - [#14161](https://github.com/NuGet/Home/issues/14161)
+
+* Update README preview to render with same font size as Visual Studio - [#14112](https://github.com/NuGet/Home/issues/14112)
+
+* NU5100 (AssemblyOutsideLibWarning): Why build is allowed but buildTransitive is not? - [#14080](https://github.com/NuGet/Home/issues/14080)
+
+* Can&#39;t copy the text from NuGet package manager gold bar - [#14074](https://github.com/NuGet/Home/issues/14074)
+
+* [DCR]: Focus shifts from Output window to Error List after every package operation, even with no error or warnings - [#11728](https://github.com/NuGet/Home/issues/11728)
```

Comment:
I'm a bit bias but given this is a DCR, and that customers have rightfully complained about it for years, can this be promoted to the What's New? At the very least, I don't agree that DCRs and "Issues fixed" should be lumped together.